packages/replay-internal/src/util/createPerformanceEntries.ts

```diff
@@ -187,22 +198,32 @@ export function getLargestContentfulPaint(metric: Metric): ReplayPerformanceEntr
   return getWebVital(metric, 'largest-contentful-paint', node);
 }
 
+function isLayoutShift(entry: PerformanceEntry | LayoutShift): entry is LayoutShift {
+  return (entry as LayoutShift).value !== undefined;
```

Comment:
Do any other `PerformanceEntry` items also have a `value` property? This seems a bit too generic

Reply:
updated!